packages/vuetify/src/labs/VNumberInput/VNumberInput.tsx

```diff
@@ -75,13 +75,14 @@ export const VNumberInput = genericComponent<VNumberInputSlots>()({
     const model = computed({
       get: () => _model.value,
       set (val) {
-        if (val === null) {
-          _model.value = null
+        if (val === null || (typeof val === 'string' && !val)) {
```

Comment:
I fully agree it looks exhaustive, but the value type is number or undefined and if we compare it without typeof there is a TS error. 
We can extend the initial type with string as well, but it will bring a bunch of refactoring.

Reply:
Oh.. I see it now. Sure, let's leave it